include/erlang_ls.hrl

```diff
@@ -149,10 +149,15 @@
 %%------------------------------------------------------------------------------
 %% Workspace Edit
 %%------------------------------------------------------------------------------
--type workspace_edit() :: #{ changes         => #{ binary() := [text_edit()]
-                                                 }
-                           , documentChanges => [text_document_edit()]
-                           }.
+
+-type document_change() :: [text_document_edit()].
+                         %% | [ text_document_edit()
```

Comment:
Please remove the commented text and create an issue so we can follow up.

Reply:
https://github.com/erlang-ls/erlang_ls/issues/365